spot-controller/package.json

```diff
@@ -29,7 +29,7 @@
         "jitsi-meet-logger": "github:jitsi/jitsi-meet-logger#5da3033d647054457597920252b3387fa43a17fa",
         "jitsi-meet-redux": "github:jitsi/jitsi-meet-redux#bdc108e3857d5649bea16ffbf60e5a367c820b5c",
         "lodash": "4.17.15",
-        "react-native-beacons-manager": "github:jitsi/react-native-beacons-manager#86847b7de89dc3c1142d68b1ce24c9ff32d734e0",
+        "react-native-beacons-manager": "github:jitsi/react-native-beacons-manager#d9e8040082570a951f3c8a7a5c96c073821841d6",
```

Comment:
To confirm, using the jitsi branch is okay? I noticed a change on master of the fork. Is that change no needed?

Reply:
Correct, it’s not needed because we now run jettify on our npm dependencies.